woocommerce/woocommerce-ios
# Order badge cuts off with two characters

Fixes #1176.

This PR tweaks the badge location, spacing and sizing for the Orders badge, based on the numbers found in the [designs file](https://www.figma.com/file/JCbo1C7AKprTKGsCFkQo0Sk9/Stats-Improvements?node-id=617%3A312)

**Before**
<img src="https://user-images.githubusercontent.com/1062444/63125809-2fc40d80-bf74-11e9-9f8c-446c1ca9aa16.png" width="350" />

**After**
<img src="https://user-images.githubusercontent.com/1062444/63171867-14074880-c002-11e9-8c0a-a66f7ded1ce0.png" width="350" />

<img src="https://user-images.githubusercontent.com/1062444/63171893-1e294700-c002-11e9-9240-90100bf3a908.png" width="350" />


### To Test
1. Run the app from develop
2. Change to a store with more than 9 new orders
3. Check out the badge
4. Check out this branch & re-run the app
5. Observe badge size

Update release notes:
